docs/tutorial/tutorial.md

```diff
@@ -315,19 +426,21 @@ function Square(props) {
 
 You'll need to change `this.props` to `props` both times it appears. Many components in your apps will be able to be written as functional components: these components tend to be easier to write and React will optimize them more in the future.
 
+<a href="https://codepen.io/brigand/pen/OmRoGV?editors=0010" target="_blank">View the current code</a>.
```

Comment:
This CodePen doesn’t work for me.

<img width="586" alt="screen shot 2017-04-26 at 12 22 32 pm" src="https://cloud.githubusercontent.com/assets/810438/25432107/106bb36a-2a7b-11e7-8d2c-f2bf3ad10fe5.png">